website/docs/users-sources/sources/social-logins/google/cloud/index.md

```diff
@@ -1,5 +1,7 @@
 ---
-title: Google
+title: Integrate with Google Cloud
```

Comment:
```suggestion
title: Google Cloud
```

Reply:
The title shown in the navigation pane on left should be the same as the title at the top of the page (consistency in what-I click-is-what-I-shoud-see)... and I don't think we should use the word "Integrate" because we want a distinction between what we are calling Integrations and what we call Sources. This docs is about using Google as a source (haha correct me if I am wrong, quite possible!).

And is it specifically OAuth, this particular setup?